Cargo.toml

```diff
@@ -11,7 +11,7 @@ description = "High resolution systems performance telemetry agent"
 [dependencies]
 async-trait = "0.1.23"
 bcc = { version = "0.0.20", optional = true }
-chashmap = "2.2.2"
+dashmap = "3.11.7"
```

Comment:
Dependencies should be kept in alphabetical order.

Reply:
I've done! cbf42edba1f8fd4cad451f26f85f9913fab19f89